sites/all/themes/humanitarianresponse/assets/sass/partials/_panel.scss

```diff
@@ -109,6 +109,22 @@
   }
 }
 
+//Events hid-contact-ref block
+.panel-pane.pane-node-field-hid-contact-ref {
```

Comment:
In this particular case it appears in a panel but it might be it doesn't for some others, please give it styles it with a more generic approach.